python/create_service_diagnostics_bundle.sh

```diff
@@ -5,7 +5,7 @@ set -eu -o pipefail
 readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 readonly DCOS_SERVICE_DIAGNOSTICS_SCRIPT_PATH="dcos-sdk-service-diagnostics/python"
 
-readonly VERSION='v0.6.0'
+readonly VERSION='v0.7.0'
```

Comment:
The version value shouldn't be updated within the PR. How to release a new version of the diagnostics tools described in CONTRIBUTING.md.